src/server/sql/user/insert_new_user.sql

```diff
@@ -3,6 +3,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 DO $$ BEGIN
```

Comment:
OED does not usually do a "DO/END" in these files unless it is a function or handles an exception. Is there a special reason it is here?

Reply:
Not sure, this was here before we touched the code.  I will remove.

--------------------------------------------------------------------
Steven Huss-Lederman, 4 years ago via PR #507   (October 6th, 2020 9:17 PM) 

make createUser more resilient, fix install bug

This tried to make createUser closer to the new createDB.
The SQL avoids an error if the user exists.
createUser now stops on errors since none should happen.
A new, optional parameer to terminateReadline allows code to supply
an exit status value where default is old value of 0.
The install script for createDB retries fixes error where it stopped on
last attempt even if it worked. It also stops if createUser fails.